apps/crn-server/test/routes/research-outputs.route.test.ts

```diff
@@ -232,6 +232,63 @@ describe('/research-outputs/ route', () => {
         expect(response.status).toBe(400);
       });
 
+      test('Should validate doi based on a regex', async () => {
```

Comment:
this is ok although maybe we could just have a single payload with all the valid fields and test the errors instead? or you can keep these tests but make sure to test the opposite scenario (with error)

Reply:
To be fair its currently possible to have both doi and rrid for instance and maybe we should actually have it documented, and if we want we can already make it impossible. 

Anyway its ok if you keep it this way just make sure to test the errors too

Reply:
@peterstarling Error cases are tested too (response2). Should I split it into separate tests?

Reply:
ahhh I see - I missed it. I wouldn't mind it that much although I think the general rule of thumb is that you should have only one expectation and also i think we usually dont do two requests in a single test so yes please - split